lib/storage.js

```diff
@@ -87,8 +87,9 @@ class Storage{
 	 * @return {Boolean}
 	 */
 	isProjectRelated(path){
-		for(const root of atom.project.rootDirectories){
-			const projectPath = root.path;
+		const rootDirectories = atom.project.rootDirectories
```

Comment:
Destructuring should be used, and a semicolon added:

~~~js
const {rootDirectories} = atom.project;
~~~